platform-service/src/unstract/platform_service/config.py

```diff
@@ -0,0 +1,94 @@
+import logging
+from logging.config import dictConfig
+
+from dotenv import load_dotenv
+from flask import Flask
+from unstract.platform_service.constants import LogLevel
+from unstract.platform_service.controller import api
+from unstract.platform_service.env import Env
+from unstract.platform_service.extensions import db
+from unstract.platform_service.utils import RemoveAccessLogTimestamp
+
+load_dotenv()
+
+
+dictConfig(
+    {
+        "version": 1,
+        "disable_existing_loggers": False,
+        "formatters": {
+            "default": {
+                "format": (
+                    "[%(asctime)s] %(levelname)s in"
+                    " %(name)s (%(module)s): %(message)s"
+                ),
+                "datefmt": "%Y-%m-%d %H:%M:%S %z",
+            },
+        },
+        "filters": {
+            "filter_timestamp": {"()": RemoveAccessLogTimestamp},
```

Comment:
@Deepak-Kesavan let's remove this filter as I suggested [here](https://github.com/Zipstack/unstract/pull/731/files#r1774476517)